src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -646,6 +646,10 @@ export class ConnectionWidget extends lifecycle.Disposable {
 				// Populate tenants (select first by default for initialization of tenant dialog)
 				await this.onAzureAccountSelected(true);
 				this._azureTenantDropdown.enable();
+				// Populate username as 'email' of selected azure account in dropdown, as username is required,
```

Comment:
"in most cases" - what are the cases it doesn't apply?

And is this username actually used anywhere? Or just for display purposes? It seems like the latter, but it'd be good to clarify that.

Reply:
Scenarios are specific to MySQL/PostgreSQL username property and how it's validated server side to provide access, and that's why the username can be different than email (optionally). I've discussed that with @rishky-msft and they indeed need this property.

Reply:
Providers that don't need Username property, go for AzureMFA authentication type.